app/javascript/src/patternflyStyles/pf4Base.css

```diff
@@ -7,6 +7,8 @@
 
 @import '~@patternfly/patternfly/components/Page/page.css';
 @import '~@patternfly/patternfly/components/Nav/nav.css';
+/* @import '~@patternfly/patternfly/components/ContextSelector/context-selector.css'; */
```

Comment:
Do we need this commented line?

Reply:
@damianpm I think that might actually be a better place for it so I put it back and removed the other include. Good catch. 
On the tests, jest still fails tests and doesn't create new snapshots. The purpose of this PR was an improvement for QE, so maybe tests need to be rewritten to use the PF classes? That's not my area, so I'm not sure.

Reply:
OK, I'll merge to a different branch, fix the tests, and then we can merge to master